text/0314-cdk-construct-hub.md

```diff
@@ -0,0 +1,310 @@
+
+Construct Hub offers an integrated search experience, allowing customers to
+easily find the right construct for their need. The built-in multi-language
+documentation helps customers understand what features are available, and learn
+how to correctly use the constructs in their chosen programming language.
+
+The Construct Hub itself is developed as an open-source construct: members
+of CDK community can freely participate in the development of the service.
+Customers with specific needs are also able to deploy a private instance of the
+Construct Hub, and integrate it with their package management solution.
+
+## FAQ
+
+### What are we launching today?
+
+Construct Hub is a new website that makes it easy for developers to
+discover, use, and share CDK constructs that encapsulate cloud infrastructure
+patterns. The community has already published hundreds of CDK constructs on the
+public registries like npmjs.com, PyPI, Maven Central, and NuGet. However those
+are distributed repositories, managed by numerous organizations, and lack
+centralized search tools sufficient for developers to easily find the cloud
+infrastructure patterns they are looking for, in the programming language they
+want to use.
+
+Construct Hub automatically indexes all CDK construct libraries published to
+the various general purpose package managers, so that developers have a single
+place to search for constructs, regardless of preferred programming language or
+provisioning engine. Developers will find constructs published by AWS, partners,
+and individual developers. CDK Constuct Hub also generates unified documentation
+for the constructs in all supported programming languages.
+
+Construct Hub is also available as a deploymable CDK construct library, so
+that enterprises and users with specific needs can host a private instance of
+the catalog, making interally developed constructs easier for teams to find.
+Just like the public instance of Construct Hub scans public repositories for
+construct libraries, the private Construct Hub instance can be configured to
+scan private repositories like AWS CodeArtifact, JFrog Artifactory, and others.
+
+### Why should I use this feature?
+
+The Construct Hub is a centralized place that helps discover and share
+custom-made constructs compatible with the AWS Cloud Development Kit (CDK), CDK
+for Kubernetes (CDK8s), or CDK for Terraform (CDK-Tf). It shortens the time it
+takes to ship cloud applications by allowing developers to discover constructs
+that address their infrastructure requirements, and provides a unified
+cross-language documentation experience to help developers integrate those
+constructs into their application.
+
+Customers can also deploy a private instance of Construct Hub to facilitate
+sharing infrastructure abstractions published to a private package registry
+within the organization.
+
+### Where can I find the Construct Hub?
+
+The public instance of Construct Hub is at: `${TBD}`.
+
```

Comment:
duplicated above

Reply:
Right up, but this is actually a relatively standard thing to put in the FAQ for some reason.